test/discharge-wizard/00-required.e2e.spec.js

```diff
@@ -42,12 +42,11 @@ module.exports = E2eHelpers.createE2eTest(
       .axeCheck('.main')
       .click('a.usa-button-primary');
 
-    client.pause(Timeouts.verySlow);
-
     // results page
     client
       .click('a.usa-button-primary')
-      .waitForElementVisible('.dw-guidance', Timeouts.verySlow)
+      .waitForElementVisible('.dw-guidance', Timeouts.slow)
+      .pause(Timeouts.verySlow)
       .axeCheck('.main');
```

Comment:
There's a PDF download occurring here to `'http://arba.army.pentagon.mil/documents/dd0293.pdf'`. It's too fast during local testing to register, but I think Jenkin's is stumbling on it.